whylog/tests/tests_config/__init__.py

```diff
@@ -7,6 +7,7 @@
 from whylog.config.parsers import RegexParserFactory
 from whylog.config.rule import RegexRuleFactory
 from whylog.teacher.user_intent import UserConstraintIntent, UserParserIntent, UserRuleIntent
+from whylog.tests.utils import path_test_files
 
 # Constraint types
 identical_constr = "identical"
```

Comment:
Maybe put somewhere const constraint types. Maybe in whylog/constraints/consts.py? 

There are now 3 files where constraint types are defined: this file, whylog/teacher/mock_outputs.py, whylog/tests/test_config/__init__.py. Maybe it would be worth to import these constraint types from one file?